src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx

```diff
@@ -17,6 +18,22 @@ export const ArtworkSidebarAuctionPolling: React.FC<Props> = ({
   relay,
   me,
 }) => {
+  const { sale, saleArtwork } = artwork
+  const isClosed = !!sale?.isClosed
+  const currentBidDisplay = saleArtwork?.currentBid?.display
+
+  const [currentBidChanged, setCurrentBidChanged] = useState(false)
+
+  const isMounted = useRef(false)
+
+  useEffect(() => {
+    if (isMounted.current) {
+      setCurrentBidChanged(true)
```

Comment:
Essentially, when the `currentBidDisplay` changes, _and_ the component has already mounted (so skipping the initial rendering), set this to true.